charts/vela-core/templates/defwithtemplate/webservice.yaml

```diff
@@ -369,6 +379,9 @@ spec:
         	// +usage=If addRevisionLabel is true, the revision label will be added to the underlying pods
         	addRevisionLabel: *false | bool
 
+        	// +usage=If addDeployVersionAnnotation is true, the deploy version annotation will be added to the underlying pods
+        	addDeployVersionAnnotation: *false | bool
```

Comment:
why do we need DeployVersion when we already have a revision?

Reply:
The component revision does not take effect as default.

Reply:
The DeployVersion is generated by the business platform. The WebService definition provides an example use case to help users know how to pass the version number to the pod.

Reply:
the business Platform should generate the DeployVersion inside the `annotations.annotations`